mantle/cmd/kola/devshell.go

```diff
@@ -284,7 +284,13 @@ loop:
 		return cmd.Run()
 	})
 	if err != nil {
-		return err
+		cmd := devshellSSH(sshConfigPath, sshKeyPath, false, "true")
+		err = cmd.Run()
+		if err != nil {
+			return errors.Wrapf(err, "failed to log in via ssh")
+		} else {
+			return fmt.Errorf("failed to log in via ssh originally, but succeeded on last attempt?")
```

Comment:
I think golint will want

```suggestion
			return errors.New("failed to log in via ssh originally, but succeeded on last attempt?")
```

But maybe just wrap the original error instead?